public/js/angular/controllers.js

```diff
@@ -254,44 +246,42 @@ angular
       $scope.weblit = weblit;
 
       $scope.wlcPoints = CONFIG.wlcPoints;
-
     }
   ])
-  .controller('competencyMediaGenController', ['$scope', '$sce',
+  .controller("competencyMediaGenController", ["$scope", "$sce",
```

Comment:
if we're rewriting this, let's use a resolver object instead of this if/else chain
```
var sources = {
  youtube: 'https://www.youtube-nocookie.com/embed/" + media.content + "?ena......
  vimeo: ...
  ...
}
return $sce.trustAsResourceUrl(sources[media]);
```